src/Common/RestApiClient.php

```diff
@@ -55,6 +55,11 @@ public function __construct($url)
         $this->client = $client;
     }
 
+    public function url()
+    {
+        return $this->client->getConfig('base_uri');
```

Comment:
Do you mean to only return the url string? Because I believe this call returns an entire object, not just the url as a string. There are additional methods you can add to grab just the url. Also, I feel like the name of the method should say more what it's trying to do... something like `getUrl()`